asset/mock.go

```diff
@@ -140,7 +140,7 @@ func virtualGenesisTxOut(newAsset *Asset) (*wire.TxOut, error) {
 // MockGroupTxBuilder.
 func virtualGenesisTx(newAsset *Asset) (*wire.MsgTx, error) {
 	var (
-		txIn *wire.TxIn
+		txIn []*wire.TxIn
```

Comment:
nit: rename to `txIns` to not shadow the name in the loop below (`for _, txIn := range txIn`)?